Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Matias Bundgaard-Nielsen <mabuni1998@gmail.com>, Mikkel Heuck <mheu@
 version = "0.2.8"
 
 [deps]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
```

Comment:
I would suggest turning this into a weakdep, otherwise anyone who installs WaveguideQED will have to download many gigabytes of CUDA runtime libraries.